documentation/polymer-templates/tutorial-template-components-in-slot.asciidoc

```diff
@@ -154,12 +164,11 @@ public class MainLayout extends PolymerTemplate<TemplateModel>
 ----
 
 [NOTE]
-The method `showRouterLayoutContent(HasElement)` in the `RouterLayout` interface 
-has default implementation so you don't need to write anything in addition to that.
-But you may reimplement it as you wish.
+The `showRouterLayoutContent(HasElement)` method in the `RouterLayout` interface has default implementations. This makes it unnecessary to write additional code, but you can re-implement it, if necessary.
```

Comment:
implementation <- singular, only one implementation.

> but you can re-implement it, if necessary.

Perhaps: but you can override and re-implement it, if necessary.

Reply:
Agreed, will update!